unpackaged/config/qa/objects/___NAMESPACE___Course_Offering__c.object

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomObject xmlns="http://soap.sforce.com/2006/04/metadata">
+    <searchLayouts>
+        <searchResultsAdditionalFields>%%%namespace%%%Course__c</searchResultsAdditionalFields>
```

Comment:
@xinnanlisf @jstvz For this & the other files - does casing matter? (Does it need to be %%%NAMESPACE%%%?)

Reply:
It does need to be in all caps -- the code that replaces the tokens is here: https://github.com/SFDO-Tooling/CumulusCI/blob/master/cumulusci/utils.py#L184

Reply:
Thanks @davisagli! We will just capitalize them so that they look consistent!